.flake8

```diff
@@ -0,0 +1,19 @@
+[flake8]
+exclude = .git,__pycache__,jetsoftime
+max-line-length = 119
+
+extend-ignore=
+  # Ignore formatting issues that a tool like black could automatically address
+  ## non-syntax indentation (visual)
+  # E122, E123, E128, E131,
+
+  ## whitespace with comments
+  E261, E262, E265,
+
+  # Ignore other issues which may or may not care about
+  ## module level import not at top of file
+  ## this is because of sys.path adding jetsoftime submodule
```

Comment:
I've had this crazy idea for a while to try and turn the randomizer into a wheel that we could just import and use instead of having to do that sys.path hack.  Though that would probably end up being more work than its' worth.

Reply:
Yeah, I was thinking about this, too. Main annoyance is version bumping. Since the `jetsoftime` code is more of an application than a library, it's pretty painful to handle without automation to bump, publish, and promote versions, at least in my past experience.

I don't think the hack's that unreasonable, makes sense, like you said, I'm kind of feeling it's more pain than worth :)